main.js

```diff
@@ -25,7 +25,7 @@ document.body.onload = function() {};
 //originally code here parsed the <Script> tags from the page to find the one that defined the PLAYER object
 //however, that's already been executed, so we can just access PLAYER - no messing around needed!
 
-var PLAYER = window.PLAYER || (unsafeWindow && unsafeWindow.PLAYER);
+var PLAYER = window.PLAYER || (typeof unsafeWindow !== 'undefined' && unsafeWindow && unsafeWindow.PLAYER);
```

Comment:
The matter of taste, but I still think that one condition is excessive
```suggestion
var PLAYER = window.PLAYER || (typeof unsafeWindow !== 'undefined' &&  unsafeWindow.PLAYER);
```

Reply:
what if `typeof unsafeWindow !== 'object'`,

Reply:
Why? Can you imaging real case?

Reply:
Oh. `===`, not `!==`.
But I agree, it makes no sense, because in JavaScript everything is an objects.